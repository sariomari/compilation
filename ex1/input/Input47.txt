;extendsarray,/*!]?*/F5739
[812
extends> ."D"array-
32
./*;})XHe?ke(?1]M*/>void+61
	:=]arrayL//array-
 -0
>intclass7
92
{{/*](
?(
!;A(!]}?]h!

y})S*/:=//oPb!]R;!?(!d	"t"{+6
nil<class8
:=//??class+1680
=<extends2
void//X)].//a}?;1Fb(ut}}
)void,extendsvoid118
]309
}nilclass)<+/*N.9/i;!(*/"l"]}[9184
/*!t*/<]"c",array. +[{void6121
+[:=11
//?5!=.7036
extends<-)void< arrayint37
.extendsstringr;67
)stringApd1-+;<)170
//]T}?!];]}];)5)
]int/*]ZL?*/,+voidarray).(x6Cz83XP,(}<stringvoid-extends:=string[/**/"j")nilextends<,	2944
][(:=><//A5]]}.l];!
stringstring;]2012
"o"-<>string<"VI"array:=;/*?6;
]!}q;}*/extends],)7559
array;<.+)<-7653
intvoid/*?
*/class92
<	Y4iSk+;//}?a
;//d
15
>/*)*/]/*Z*/,//?}7};;(extendsextends.arrayextends83
6467
>.nil)+{array//l!}.[9n;(]J
class655
]6
(18
GwQB543
]/*!;;(*/,{[class{6660
 "d"[,[)nil{void)/*}
2);Q?;}V)Q}?;)
*/=intint[arrayextends string{:=-stringstring)string//;9!?>p{extends8052
,class<[extends<<Fint]6337
>nil97
>:=//k2t;)3
="":=class(>=={{,>"iy""J")nil>extends{>.[]89
KY(<}<[;>-/*;
*/]/*1]6;();8S*/;+>)"VcPHI"./*!];!
*/++=[:=int961
void9
,nilarray-=+array//W//b;?5
"VqGtEhzJJb"555
 "vw",	]-:=,;class+(nil:=/*(9??(6;x*/array{:=array""1
]	 41
array//]4}c2n3)L;;;/?;(;
void886
nil.,"INb"-.int+Zvoid>>-;(>8040
<-;6
voidarrayxiN3545nil38
94

arraynil:=/*}}*/;;/*;W(si;?*/}[:=t49ZgXFv7
 extendsnilint):=class:=]=-.//{  classy<extends6461
//!ak
+:=int,,classarray5707
=nil/*9
?!1)89;
)0}*///?}S}!;x!}}?}M;?
Fntclass//H}]; class}]-/*(1}(]M!]?}9;?*/,]extendsarraystringextends:=[,<)]{//<[583
;/*]
]f;A}!lN!*/class/*?*/class""extends<,:=int850
,:=[{=1
void222
>/*]
*/extends(int<:=,/*f]A?(z*/>}class
(->int}extends+=+5
""
-nil""int===string=/*(D)]
(*/516
8366
<);:="FABb"nil)extendsstring"vg".{.//e]r?!.!}4};
;	/*?*/]cJt{nil65
1123
 <{=HKnoum5dd0MR4.void3
extends
//{{8
=[:=ZUnilnil156
=/*(;!yw)?U?
?*/696
class:=""/*?*/:=voidextends3524
9048
{=,}string3538
+.][.string"MF"6
2
-stringnilint"R"),:=class5

.59
>{.==int916
string+/**/}=string)//;!?j?f287
class.=;9418
 87
extendsstring83
array<:=<21
-54
//(.{N(}!??3.extends995
i9M0359 163
]extends+nil
[<57
 intb6b617(extends} "kQJu"""array,+void];;;>2
"nOhU"1
int238
<int+intclassarray	<=]}intextends713
838
]class27
//!.
nil[extends<-+arrayarraynil/*;}n
*/."E"nil)>/*w*/;<
,//
]extends77
/*
U
!
H)?!
*/void[ 8321
>arraystringextendsextendsclassextends{array-array254
array0
]/**/class:=string(array6544
:=(<.=-+void5360
(761
90
//}};9]],(string
"F"string+array."B"	/*?
*/int=//]2))/?6?{
{) 5
"Ia"tw0)void(:=.348
>>stringarray:=""(+0
void>9
(nilha/*?;*/(void)+318
arrayintarray-""/*?;42}?*/=stringclassclassclass:=5
[){("G"classstringextends(string41
[--})5734
:=[extends9844
;821
32
array}2
;>,[class67
]27
int3
63
string1
4
string"mhl"<-int=5472
3
}	//?}
;3641
extends5
;[>class-void604
[{9475
=(<]string[)],93
	7
=string"")""-),<};-/**/Gextends)void:=int/**/stringvoidintF85k6746
array:=694
)5
intvoidclass=935
452
)}[=d>;class classclass840
