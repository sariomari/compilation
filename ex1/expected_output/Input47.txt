SEMICOLON[1,1]
ID(extendsarray)[1,2]
COMMA[1,14]
ID(F5739)[2,6]
LBRACK[3,1]
INT(812)[3,2]
EXTENDS[4,1]
GT[4,8]
DOT[4,10]
STRING("D")[4,11]
ARRAY[4,14]
MINUS[4,19]
INT(32)[5,1]
DOT[6,1]
GT[7,15]
TYPE_VOID[7,16]
PLUS[7,20]
INT(61)[7,21]
ASSIGN[8,2]
RBRACK[8,4]
ID(arrayL)[8,5]
ARRAY[9,1]
MINUS[9,6]
MINUS[10,2]
INT(0)[10,3]
GT[11,1]
ID(intclass7)[11,2]
INT(92)[12,1]
LBRACE[13,1]
LBRACE[13,2]
ASSIGN[19,3]
STRING("t")[20,2]
LBRACE[20,5]
PLUS[20,6]
INT(6)[20,7]
NIL[21,1]
LT[21,4]
ID(class8)[21,5]
ASSIGN[22,1]
CLASS[23,1]
PLUS[23,6]
INT(1680)[23,7]
EQ[24,1]
LT[24,2]
ID(extends2)[24,3]
TYPE_VOID[25,1]
RBRACK[26,1]
DOT[26,2]
RPAREN[27,1]
TYPE_VOID[27,2]
COMMA[27,6]
ID(extendsvoid118)[27,7]
RBRACK[28,1]
INT(309)[28,2]
RBRACE[29,1]
ID(nilclass)[29,2]
RPAREN[29,10]
LT[29,11]
PLUS[29,12]
STRING("l")[30,4]
RBRACK[30,7]
RBRACE[30,8]
LBRACK[30,9]
INT(9184)[30,10]
LT[32,5]
RBRACK[32,6]
STRING("c")[32,7]
COMMA[32,10]
ARRAY[32,11]
DOT[32,16]
PLUS[32,18]
LBRACK[32,19]
LBRACE[32,20]
ID(void6121)[32,21]
PLUS[33,1]
LBRACK[33,2]
ASSIGN[33,3]
INT(11)[33,5]
EQ[35,1]
DOT[35,2]
INT(7036)[35,3]
EXTENDS[36,1]
LT[36,8]
MINUS[36,9]
RPAREN[36,10]
TYPE_VOID[36,11]
LT[36,15]
ID(arrayint37)[36,17]
DOT[37,1]
ID(extendsstringr)[37,2]
SEMICOLON[37,16]
INT(67)[37,17]
RPAREN[38,1]
ID(stringApd1)[38,2]
MINUS[38,12]
PLUS[38,13]
SEMICOLON[38,14]
LT[38,15]
RPAREN[38,16]
INT(170)[38,17]
RBRACK[40,1]
TYPE_INT[40,2]
COMMA[40,13]
PLUS[40,14]
ID(voidarray)[40,15]
RPAREN[40,24]
DOT[40,25]
LPAREN[40,26]
ID(x6Cz83XP)[40,27]
COMMA[40,35]
LPAREN[40,36]
RBRACE[40,37]
LT[40,38]
ID(stringvoid)[41,1]
MINUS[41,11]
EXTENDS[41,12]
ASSIGN[41,19]
TYPE_STRING[41,21]
LBRACK[41,27]
STRING("j")[41,32]
RPAREN[41,35]
ID(nilextends)[41,36]
LT[41,46]
COMMA[41,47]
INT(2944)[41,49]
RBRACK[42,1]
LBRACK[42,2]
LPAREN[42,3]
ASSIGN[42,4]
GT[42,6]
LT[42,7]
ID(stringstring)[43,1]
SEMICOLON[43,13]
RBRACK[43,14]
INT(2012)[43,15]
STRING("o")[44,1]
MINUS[44,4]
LT[44,5]
GT[44,6]
TYPE_STRING[44,7]
LT[44,13]
STRING("VI")[44,14]
ARRAY[44,18]
ASSIGN[44,23]
SEMICOLON[44,25]
EXTENDS[45,9]
RBRACK[45,16]
COMMA[45,17]
RPAREN[45,18]
INT(7559)[45,19]
ARRAY[46,1]
SEMICOLON[46,6]
LT[46,7]
DOT[46,8]
PLUS[46,9]
RPAREN[46,10]
LT[46,11]
MINUS[46,12]
INT(7653)[46,13]
ID(intvoid)[47,1]
ID(class92)[49,3]
LT[50,1]
ID(Y4iSk)[50,3]
PLUS[50,8]
SEMICOLON[50,9]
SEMICOLON[51,1]
INT(15)[52,1]
GT[53,1]
RBRACK[53,7]
COMMA[53,13]
LPAREN[54,1]
ID(extendsextends)[54,2]
DOT[54,16]
ID(arrayextends83)[54,17]
INT(6467)[55,1]
GT[56,1]
DOT[56,2]
NIL[56,3]
RPAREN[56,6]
PLUS[56,7]
LBRACE[56,8]
ARRAY[56,9]
ID(class655)[58,1]
RBRACK[59,1]
INT(6)[59,2]
LPAREN[60,1]
INT(18)[60,2]
ID(GwQB543)[61,1]
RBRACK[62,1]
COMMA[62,10]
LBRACE[62,11]
LBRACK[62,12]
CLASS[62,13]
LBRACE[62,18]
INT(6660)[62,19]
STRING("d")[63,2]
LBRACK[63,5]
COMMA[63,6]
LBRACK[63,7]
RPAREN[63,8]
NIL[63,9]
LBRACE[63,12]
TYPE_VOID[63,13]
RPAREN[63,17]
EQ[68,3]
ID(intint)[68,4]
LBRACK[68,10]
ID(arrayextends)[68,11]
TYPE_STRING[68,24]
LBRACE[68,30]
ASSIGN[68,31]
MINUS[68,33]
ID(stringstring)[68,34]
RPAREN[69,1]
TYPE_STRING[69,2]
GT[70,1]
ID(p)[70,2]
LBRACE[70,3]
ID(extends8052)[70,4]
COMMA[71,1]
CLASS[71,2]
LT[71,7]
LBRACK[71,8]
EXTENDS[71,9]
LT[71,16]
LT[71,17]
ID(Fint)[71,18]
RBRACK[71,22]
INT(6337)[71,23]
GT[72,1]
ID(nil97)[72,2]
GT[73,1]
ASSIGN[73,2]
INT(3)[74,1]
EQ[75,1]
STRING("")[75,2]
ASSIGN[75,4]
CLASS[75,6]
LPAREN[75,11]
GT[75,12]
EQ[75,13]
EQ[75,14]
LBRACE[75,15]
LBRACE[75,16]
COMMA[75,17]
GT[75,18]
STRING("iy")[75,19]
STRING("J")[75,23]
RPAREN[75,26]
NIL[76,1]
GT[76,4]
EXTENDS[76,5]
LBRACE[76,12]
GT[76,13]
DOT[76,14]
LBRACK[76,15]
RBRACK[76,16]
INT(89)[76,17]
ID(KY)[77,1]
LPAREN[77,3]
LT[77,4]
RBRACE[77,5]
LT[77,6]
LBRACK[77,7]
SEMICOLON[77,8]
GT[77,9]
MINUS[77,10]
RBRACK[78,3]
SEMICOLON[79,11]
PLUS[80,1]
GT[80,2]
RPAREN[80,3]
STRING("VcPHI")[80,4]
DOT[80,11]
PLUS[81,3]
PLUS[81,4]
EQ[81,5]
LBRACK[81,6]
ASSIGN[81,7]
ID(int961)[81,9]
ID(void9)[82,1]
COMMA[83,1]
ID(nilarray)[83,2]
MINUS[83,10]
EQ[83,11]
PLUS[83,12]
ARRAY[83,13]
ID(W)[84,1]
INT(5)[85,1]
STRING("VqGtEhzJJb")[86,1]
INT(555)[86,13]
STRING("vw")[87,2]
COMMA[87,6]
RBRACK[87,8]
MINUS[87,9]
ASSIGN[87,10]
COMMA[87,12]
SEMICOLON[87,13]
CLASS[87,14]
PLUS[87,19]
LPAREN[87,20]
NIL[87,21]
ASSIGN[87,24]
ARRAY[88,8]
LBRACE[88,13]
ASSIGN[88,14]
ARRAY[88,16]
STRING("")[88,21]
INT(1)[88,23]
RBRACK[89,1]
INT(41)[90,2]
ARRAY[91,1]
ID(void886)[92,1]
NIL[93,1]
DOT[93,4]
COMMA[93,5]
STRING("INb")[93,6]
MINUS[93,11]
DOT[93,12]
TYPE_INT[93,13]
PLUS[93,16]
ID(Zvoid)[93,17]
GT[93,22]
GT[93,23]
MINUS[93,24]
SEMICOLON[93,25]
LPAREN[93,26]
GT[93,27]
INT(8040)[93,28]
LT[94,1]
MINUS[94,2]
SEMICOLON[94,3]
INT(6)[94,4]
ID(voidarrayxiN3545nil38)[95,1]
INT(94)[96,1]
ID(arraynil)[98,1]
ASSIGN[98,9]
SEMICOLON[99,7]
SEMICOLON[99,8]
RBRACE[99,20]
LBRACK[99,21]
ASSIGN[99,22]
ID(t49ZgXFv7)[99,24]
ID(extendsnilint)[100,2]
RPAREN[100,15]
ASSIGN[100,16]
CLASS[100,18]
ASSIGN[100,23]
RBRACK[100,25]
EQ[100,26]
MINUS[100,27]
DOT[100,28]
LBRACE[101,1]
ID(classy)[101,4]
LT[101,10]
ID(extends6461)[101,11]
PLUS[103,1]
ASSIGN[103,2]
TYPE_INT[103,4]
COMMA[103,7]
COMMA[103,8]
ID(classarray5707)[103,9]
EQ[104,1]
NIL[104,2]
ID(Fntclass)[109,1]
CLASS[110,2]
RBRACE[110,7]
RBRACK[110,8]
MINUS[110,9]
COMMA[111,4]
RBRACK[111,5]
ID(extendsarraystringextends)[111,6]
ASSIGN[111,31]
LBRACK[111,33]
COMMA[111,34]
LT[111,35]
RPAREN[111,36]
RBRACK[111,37]
LBRACE[111,38]
LT[112,1]
LBRACK[112,2]
INT(583)[112,3]
SEMICOLON[113,1]
CLASS[115,9]
CLASS[115,19]
STRING("")[116,1]
EXTENDS[116,3]
LT[116,10]
COMMA[116,11]
ASSIGN[116,12]
ID(int850)[116,14]
COMMA[117,1]
ASSIGN[117,2]
LBRACK[117,4]
LBRACE[117,5]
EQ[118,1]
INT(1)[118,2]
ID(void222)[119,1]
GT[120,1]
EXTENDS[122,3]
LPAREN[122,10]
TYPE_INT[122,11]
LT[122,14]
ASSIGN[122,15]
COMMA[122,17]
GT[122,28]
RBRACE[122,29]
CLASS[122,30]
LPAREN[123,1]
MINUS[123,2]
GT[124,1]
TYPE_INT[124,2]
RBRACE[124,5]
EXTENDS[124,6]
PLUS[124,13]
EQ[124,14]
PLUS[124,15]
INT(5)[124,16]
STRING("")[125,1]
MINUS[126,1]
NIL[126,2]
STRING("")[126,5]
TYPE_INT[126,7]
EQ[126,10]
EQ[126,11]
EQ[126,12]
TYPE_STRING[126,13]
EQ[126,19]
INT(516)[128,4]
INT(8366)[129,1]
LT[130,1]
RPAREN[130,2]
SEMICOLON[130,3]
ASSIGN[130,4]
STRING("FABb")[130,6]
NIL[130,12]
RPAREN[130,15]
ID(extendsstring)[130,16]
STRING("vg")[130,29]
DOT[130,33]
LBRACE[130,34]
DOT[130,35]
SEMICOLON[131,1]
RBRACK[131,8]
ID(cJt)[131,9]
LBRACE[131,12]
ID(nil65)[131,13]
INT(1123)[132,1]
LT[133,2]
LBRACE[133,3]
EQ[133,4]
ID(HKnoum5dd0MR4)[133,5]
DOT[133,18]
ID(void3)[133,19]
EXTENDS[134,1]
LBRACE[136,1]
LBRACE[136,2]
INT(8)[136,3]
EQ[137,1]
LBRACK[137,2]
ASSIGN[137,3]
ID(ZUnilnil156)[137,5]
EQ[138,1]
INT(696)[140,4]
CLASS[141,1]
ASSIGN[141,6]
STRING("")[141,8]
ASSIGN[141,15]
ID(voidextends3524)[141,17]
INT(9048)[142,1]
LBRACE[143,1]
EQ[143,2]
COMMA[143,3]
RBRACE[143,4]
ID(string3538)[143,5]
PLUS[144,1]
DOT[144,2]
RBRACK[144,3]
LBRACK[144,4]
DOT[144,5]
TYPE_STRING[144,6]
STRING("MF")[144,12]
INT(6)[144,16]
INT(2)[145,1]
MINUS[146,1]
ID(stringnilint)[146,2]
STRING("R")[146,14]
RPAREN[146,17]
COMMA[146,18]
ASSIGN[146,19]
ID(class5)[146,21]
DOT[148,1]
INT(59)[148,2]
GT[149,1]
LBRACE[149,2]
DOT[149,3]
EQ[149,4]
EQ[149,5]
ID(int916)[149,6]
TYPE_STRING[150,1]
PLUS[150,7]
RBRACE[150,12]
EQ[150,13]
TYPE_STRING[150,14]
RPAREN[150,20]
ID(f287)[151,1]
CLASS[152,1]
DOT[152,6]
EQ[152,7]
SEMICOLON[152,8]
INT(9418)[152,9]
INT(87)[153,2]
ID(extendsstring83)[154,1]
ARRAY[155,1]
LT[155,6]
ASSIGN[155,7]
LT[155,9]
INT(21)[155,10]
MINUS[156,1]
INT(54)[156,2]
DOT[158,1]
ID(extends995)[158,2]
ID(i9M0359)[160,1]
INT(163)[160,9]
RBRACK[161,1]
EXTENDS[161,2]
PLUS[161,9]
NIL[161,10]
LBRACK[162,1]
LT[162,2]
INT(57)[162,3]
ID(intb6b617)[163,2]
LPAREN[163,11]
EXTENDS[163,12]
RBRACE[163,19]
STRING("kQJu")[163,21]
STRING("")[163,27]
ARRAY[163,29]
COMMA[163,34]
PLUS[163,35]
TYPE_VOID[163,36]
RBRACK[163,40]
SEMICOLON[163,41]
SEMICOLON[163,42]
SEMICOLON[163,43]
GT[163,44]
INT(2)[163,45]
STRING("nOhU")[164,1]
INT(1)[164,7]
ID(int238)[165,1]
LT[166,1]
TYPE_INT[166,2]
PLUS[166,5]
ID(intclassarray)[166,6]
LT[166,20]
EQ[166,21]
RBRACK[166,22]
RBRACE[166,23]
ID(intextends713)[166,24]
INT(838)[167,1]
RBRACK[168,1]
ID(class27)[168,2]
NIL[170,1]
LBRACK[170,4]
EXTENDS[170,5]
LT[170,12]
MINUS[170,13]
PLUS[170,14]
ID(arrayarraynil)[170,15]
DOT[172,3]
STRING("E")[172,4]
NIL[172,7]
RPAREN[172,10]
GT[172,11]
SEMICOLON[172,17]
LT[172,18]
COMMA[174,1]
RBRACK[175,1]
ID(extends77)[175,2]
TYPE_VOID[180,3]
LBRACK[180,7]
INT(8321)[180,9]
GT[181,1]
ID(arraystringextendsextendsclassextends)[181,2]
LBRACE[181,39]
ARRAY[181,40]
MINUS[181,45]
ID(array254)[181,46]
ID(array0)[182,1]
RBRACK[183,1]
CLASS[183,6]
ASSIGN[183,11]
TYPE_STRING[183,13]
LPAREN[183,19]
ID(array6544)[183,20]
ASSIGN[184,1]
LPAREN[184,3]
LT[184,4]
DOT[184,5]
EQ[184,6]
MINUS[184,7]
PLUS[184,8]
ID(void5360)[184,9]
LPAREN[185,1]
INT(761)[185,2]
INT(90)[186,1]
RBRACK[188,1]
COMMA[188,2]
LPAREN[188,3]
TYPE_STRING[188,4]
STRING("F")[189,1]
TYPE_STRING[189,4]
PLUS[189,10]
ARRAY[189,11]
DOT[189,16]
STRING("B")[189,17]
TYPE_INT[190,3]
EQ[190,6]
LBRACE[191,1]
RPAREN[191,2]
INT(5)[191,4]
STRING("Ia")[192,1]
ID(tw0)[192,5]
RPAREN[192,8]
TYPE_VOID[192,9]
LPAREN[192,13]
ASSIGN[192,14]
DOT[192,16]
INT(348)[192,17]
GT[193,1]
GT[193,2]
ID(stringarray)[193,3]
ASSIGN[193,14]
STRING("")[193,16]
LPAREN[193,18]
PLUS[193,19]
INT(0)[193,20]
TYPE_VOID[194,1]
GT[194,5]
INT(9)[194,6]
LPAREN[195,1]
ID(nilha)[195,2]
LPAREN[195,13]
TYPE_VOID[195,14]
RPAREN[195,18]
PLUS[195,19]
INT(318)[195,20]
ID(arrayintarray)[196,1]
MINUS[196,14]
STRING("")[196,15]
EQ[196,27]
ID(stringclassclassclass)[196,28]
ASSIGN[196,49]
INT(5)[196,51]
LBRACK[197,1]
RPAREN[197,2]
LBRACE[197,3]
LPAREN[197,4]
STRING("G")[197,5]
ID(classstringextends)[197,8]
LPAREN[197,26]
ID(string41)[197,27]
LBRACK[198,1]
MINUS[198,2]
MINUS[198,3]
RBRACE[198,4]
RPAREN[198,5]
INT(5734)[198,6]
ASSIGN[199,1]
LBRACK[199,3]
ID(extends9844)[199,4]
SEMICOLON[200,1]
INT(821)[200,2]
INT(32)[201,1]
ARRAY[202,1]
RBRACE[202,6]
INT(2)[202,7]
SEMICOLON[203,1]
GT[203,2]
COMMA[203,3]
LBRACK[203,4]
ID(class67)[203,5]
RBRACK[204,1]
INT(27)[204,2]
ID(int3)[205,1]
INT(63)[206,1]
ID(string1)[207,1]
INT(4)[208,1]
TYPE_STRING[209,1]
STRING("mhl")[209,7]
LT[209,12]
MINUS[209,13]
TYPE_INT[209,14]
EQ[209,17]
INT(5472)[209,18]
INT(3)[210,1]
RBRACE[211,1]
SEMICOLON[212,1]
INT(3641)[212,2]
ID(extends5)[213,1]
SEMICOLON[214,1]
LBRACK[215,1]
GT[215,2]
CLASS[215,3]
MINUS[215,8]
ID(void604)[215,9]
LBRACK[216,1]
LBRACE[216,2]
INT(9475)[216,3]
EQ[217,1]
LPAREN[217,2]
LT[217,3]
RBRACK[217,4]
TYPE_STRING[217,5]
LBRACK[217,11]
RPAREN[217,12]
RBRACK[217,13]
COMMA[217,14]
INT(93)[217,15]
INT(7)[219,1]
EQ[220,1]
TYPE_STRING[220,2]
STRING("")[220,8]
RPAREN[220,10]
STRING("")[220,11]
MINUS[220,13]
RPAREN[220,14]
COMMA[220,15]
LT[220,16]
RBRACE[220,17]
SEMICOLON[220,18]
MINUS[220,19]
ID(Gextends)[220,24]
RPAREN[220,32]
TYPE_VOID[220,33]
ASSIGN[220,37]
TYPE_INT[220,39]
ID(stringvoidintF85k6746)[220,46]
ARRAY[221,1]
ASSIGN[221,6]
INT(694)[221,8]
RPAREN[222,1]
INT(5)[222,2]
ID(intvoidclass)[223,1]
EQ[223,13]
INT(935)[223,14]
INT(452)[224,1]
RPAREN[225,1]
RBRACE[225,2]
LBRACK[225,3]
EQ[225,4]
ID(d)[225,5]
GT[225,6]
SEMICOLON[225,7]
CLASS[225,8]
ID(classclass840)[225,14]